lms/djangoapps/teams/models.py

```diff
@@ -15,6 +15,7 @@ class CourseTeam(models.Model):
     """This model represents team related info."""
 
     team_id = models.CharField(max_length=255, unique=True)
+    discussion_id = models.CharField(max_length=255, unique=True)
```

Comment:
On my development environment, an index is added when the field is marked unique.